lib/diaspora/message_renderer.rb

```diff
@@ -50,7 +50,7 @@ def escape
       end
 
       def strip_markdown
-        renderer = Redcarpet::Markdown.new Redcarpet::Render::StripDown, options[:markdown_options]
+        renderer = Redcarpet::Markdown.new Redcarpet::Render::StripDown, options[:stripdown_options]
```

Comment:
Why does this need special `stripdown_options` without `footnotes`? So text-only doesn't support footnotes?

I think the reason for this should be commented here. And I'd also prefer still using `options[:markdown_options]` but just explicitly dropping `footnotes` from it (or set it to `false`?), so there is still only one `markdown_options` in `DEFAULTS` and if `markdown_options` needs to be changed it only needs to be changed once. Otherwise maybe in the future only `markdown_options` gets updated and `stripdown_options` is forgotten.